YoutubeKit/Player/YTSwiftyPlayerHTML.swift

```diff
@@ -44,6 +43,13 @@ class YTSwiftyPlayerHTML {
         }
         window.setInterval(updateTime, 500);
         });
+        $(document).ready(function(){
+        var meta = document.createElement('meta');
+        meta.setAttribute('name', 'viewport');
+        meta.setAttribute('content', 'width=device-width');
+        meta.setAttribute('initial-scale', '1');
+        document.getElementsByTagName('head')[0].appendChild(meta);
+        })
```

Comment:
OK I understand this and it should be fine if it works, but there's a lot of issues for this JS code:

1. I'm not sure why this is working, but you're calling jQuery. Do not use any external libraries. Instead:

```
document.addEventListener("DOMContentLoaded", function(event) { 
  // your code
});
```

https://stackoverflow.com/questions/799981/document-ready-equivalent-without-jquery
https://developer.mozilla.org/en-US/docs/Web/API/Document/DOMContentLoaded_event

2. `let` instead of `var`. I think in this case `const` is far better.

```
        const meta = document.createElement('meta');
```

3. Trailing semicolon `;`

```
        });
```

4. Move the code to the bottom of the script tag.

Reply:
I have tried several ways to re-write js but Youtube title still keeps big - fit to screen width.

![Simulator Screen Shot - iPhone 5s - 2019-08-20 at 11 33 15](https://user-images.githubusercontent.com/7288035/64309328-68c92e00-cfc6-11e9-9258-39b3c32a2d4f.png)

Maybe my previous js doesn't work, so it doesn't affect to title.
As I understand, `<meta name="viewport" content="width=device-width,initial-scale=1">` instruct browser to scale to fit with screen width, so title becomes bigger seems a right result. 

Do you think that removing viewport meta is worth to be able to handle fullscreen and volume button easier?

I will keep this PR purpose is to show fullscreen, volume button; and remove changes related to scaling, please consider!

Reply:
Nope, that is absolutely the problem of YouTube player a.k.a. Google. As far as I can see, they are not considering to embed the inline player to the mobile viewport and that's why. But of course, we're embedding the video to the mobile so it is almost always a must to specify viewport otherwise it doesn't fit or scale, like the fullscreen button.